client/state/data-layer/wpcom/sites/automated-transfer/status/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { delay } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { AUTOMATED_TRANSFER_STATUS_REQUEST } from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import {
+	getAutomatedTransferStatus,
+	setAutomatedTransferStatus,
+} from 'state/automated-transfer/actions';
+
+export const requestStatus = ( { dispatch }, action ) => {
+	const { siteId } = action;
+
+	dispatch( http( {
+		method: 'GET',
+		path: `/sites/${ siteId }/automated-transfers/status`,
+		apiVersion: '1',
+	}, action ) );
+};
+
+export const receiveStatus = ( { dispatch }, { siteId }, next, { status, uploaded_plugin_slug } ) => {
```

Comment:
The `next` param is no longer passed to `dispatchRequest` handlers since #17061 was merged. Are you sure this code works and that the `{ status, uploaded_plugin_slug }` have the expected values? I'd expect the destructuring to crash, as the parameter is `undefined`.

Reply:
Nice spot @jsnajdr. PR #17529 where this is working is based on a commit from 2nd August! I'll remove the `next` params.